### ISSUE TITLE
Cannot find .d.ts file in subdirectory

### ISSUE THREAD

**Issue by emlai** [pzavolinsky/ts-unused-exports#29]
x/a.d.ts:
```ts
export type A = number
```

b.ts:
```ts
import { A } from './x/a'
export const a: A = 0
```

tsconfig.json:
```json
{
  "include": [
    "./**/*"
  ]
}
```

Output of `ts-unused-exports ./tsconfig.json`:
```
Cannot find module 'x/a'.
  I've tried the following paths and none of them works:
  - x/a.ts
  - x/a.tsx
  - x/a/index.ts
  - x/a/index.tsx
  - x/a.js
  - x/a (only to skip it later)
```

This doesn't happen if `a.d.ts` is in `./` or if it's named `a.ts`.

**Comment by mrseanryan** [comment]
hi @emlai  this issues appears to be already fixed (at least in version `3.0.3`).

There is a PR to add tests to cover this case:

https://github.com/pzavolinsky/ts-unused-exports/pull/70

**Comment by mrseanryan** [comment]
Closing - appears to be resolved.
